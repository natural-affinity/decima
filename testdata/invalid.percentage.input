decima -p PERCENTAGE 500
