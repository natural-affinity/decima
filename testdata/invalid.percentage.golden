main.go:51: invalid percentage: key: "--percent" failed type conversion: strconv.ParseFloat: parsing "PERCENTAGE": invalid syntax
